app/src/components/composites/Navbar/styled.ts

```diff
@@ -0,0 +1,34 @@
+export const boxProps = {
```

Comment:
fix styles all over the app
```suggestion
export const StyledBox = styled(Box)({
  display: 'flex',
  flexDirection: 'column',
  alignItems: 'center',
  height: '100vh',
  borderRight: '1px solid #0b31be',
  outline: '1px solid #0b31be',
});
```

Reply:
fix all over the app